modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/global-js/AttributeFields.tsx

```diff
@@ -147,7 +155,10 @@ export default function AttributeFields({
 					)}
 
 					<div className="d-flex justify-content-between">
-						<label htmlFor={valueId}>
+						<label
+							className={`${disabled ? ' disabled' : ''}`}
```

Comment:
This can just be: `className={disabled ? 'disabled' : ''}`

Reply:
Right, changing that, thanks.